packages/http/lib/decorators.tsp

```diff
@@ -39,6 +39,49 @@ model HeaderOptions {
  */
 extern dec header(target: ModelProperty, headerNameOrOptions?: string | HeaderOptions);
 
+/**
+ * Cookie Options.
+ */
+model CookieOptions {
+  /**
+   * Name in the cookie.
+   */
+  name?: string;
+
+  /**
+   * If false the value will be joined with `,`.
+   *
+   *  | Style | Explode | Uri Template | Primitive value id = 5 | Array id = [3, 4, 5] | Object id = {"role": "admin", "firstName": "Alex"} |
+   *  | ----- | ------- | -------------| ---------------------- | -------------------- | -------------------------------------------------- |
+   *  | form  | true    | `         `  | `Cookie: id=5`         |                      |                                                    |
+   *  | form  | false   | `id={id}`    | `Cookie: id=5`         | `Cookie: id=3,4,5   `| `Cookie: id=role,admin,firstName,Alex`             |
+   *
+   */
+  explode?: boolean;
```

Comment:
looking at the openapi table, I am wondering if we do want explode at all as an option. It feels like it does nothing but limit usage no?

Reply:
I also thought that explode might not be necessary, but I added it just in case it is needed as a schema.

In relation to this, I'm wondering if style is necessary. In the case of cookies, it seems that only `form` can be specified, but in the same way, style is also provided for header, where only `simple` can be specified.

Reply:
yeah I would omit style and explode for now. Our default for explode in query and path are different from openapi(we use false, they use true) so it could always be added later if figured out it was neeeded